config/vlan.py

```diff
@@ -316,9 +316,7 @@ def add_vlan_member(db, vid, port, untagged, multiple, except_flag):
             elif not is_port:
                 port_data = config_db.get_entry('PORTCHANNEL',port)
 
-            if "mode" not in port_data: 
-                ctx.fail("{} is in routed mode!\nUse switchport mode command to change port mode".format(port))
-            else:
+            if "mode" in port_data:
                 existing_mode = port_data["mode"]
             
             if existing_mode == "routed":
```

Comment:
If "mode" not in `port_data`, the `existing_mode` should be undefined. 
Why is it not caught by build or test?